### PR TITLE
BDOG-632: improve logging of upload key / reference

### PR DESCRIPTION
**Pull request by nigelhp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,14 +78,15 @@ The service must also provide a callbackUrl for asynchronous notification of the
...
+Session-ID / Request-ID headers are used to link the file with the user's journey.
 
 *Note:* If you are using `[http-verbs](https://github.com/hmrc/http-verbs)` to call Upscan, all the headers will be set automatically
 (See: [HttpVerb.scala](https://github.com/hmrc/http-verbs/blob/2807dc65f64009bd7ce1f14b38b356e06dd23512/src/main/scala/uk/gov/hmrc/http/HttpVerb.scala#L53))
 
 The service replies with a pre-filled template for the upload of the file.
-The JSON response also contains a globally unique file reference for the upload. This reference can be used by the Upscan service team to view the progress and result of the journey through the different Upscan components. The consuming service can use this reference to correlate the subsequent upload result with this upscan initiation.
-
+The JSON response contains a globally unique identifier for the upload (known as both _reference_ and _key_).
+This identifier can be used by the Upscan service team to view the progress and result of the journey through the different Upscan components.
```
Comment (chotai) [inline README.md:88]: Is it worth mentioning something to the effect of - if there's an issue and that requires further investigation we require the reference.

File: conf/application.conf [hunk 2]
```diff
@@ -96,7 +96,7 @@ callbackValidation.allowedProtocols = "https"
 global.file.size.limit = 104857600
 
 auditing {
-  enabled = true
+  enabled = false
```
Comment (chotai) [inline conf/application.conf:99]: Should auditing be enabled?

**Review by chotai — DISMISSED** [review]

File: conf/application.conf (hunk 2 above)
Comment (nigelhp) [reply conf/application.conf:99]: Auditing is enabled for real environments via app-config-base see: https://github.com/hmrc/app-config-base/blob/8abf8c3754008d7c0be7ac646439ae5a9b2dbf38/upscan-initiate.conf#L9

This just disables it for local development to stop spamming the log with connection failures when a local datastream is not running.

File: README.md (hunk 1 above)
Comment (nigelhp) [reply README.md:88]: Yes - added

**Review by chotai — APPROVED** [review]

**Pull request merged**